src/openms/source/ANALYSIS/MRM/ReactionMonitoringTransition.cpp

```diff
@@ -284,4 +284,36 @@ namespace OpenMS
     library_intensity_ = intensity;
   }
 
+  bool ReactionMonitoringTransition::isDetectingTransition() const
+  {
+    bool detecting = true;
+    if (this->getMetaValue("detecting_transition").toString() == "0") {detecting = false;}
```

Comment:
The function setMetaValue accepts boolean values but stores them as integers. Reading them back from TraML generates integer values. The CSV assay lists contain integers. How should we best deal with this? Is storing of the values as OpenMS::String preferred?